projects/ion/src/lib/modal/component/modal.component.spec.ts

```diff
@@ -196,4 +196,161 @@ describe('IonModalComponent', () => {
       fixture.nativeElement.querySelector('.modal-container').style.width;
     expect(modalElement).toBe(`${modalConfig.width}px`);
   });
+
+  describe('IonModalComponent - Header left button', () => {
+    it('should not be rendered as default', () => {
+      expect(screen.queryByTestId('btn-voltar')).not.toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.queryByTestId('header-button')).not.toBeInTheDocument();
```

that wouldn't be it?

Reply:
I need to select the button element inside the ion-button, otherwise jest fires a click that doesn't emit the event.